plugins/editor/class.editor.plugin.php

```diff
@@ -1096,6 +1096,13 @@ protected function cacheAttachedMedia($Sender) {
 
         $DiscussionID = null;
         $Comments = $Sender->data('Comments');
+        if ($answers = $Sender->data('Answers')) {
+            $commentsArray = $Comments->resultObject();
```

Comment:
For the record, a method named `resultObject()` that returns an array is the work of the devil.